claim-check/code-samples/sample-2/client-consumer/Program.cs

```diff
@@ -19,13 +18,13 @@ public class Program
 
         public static async Task Main(string[] args)
         {
-            IConsumer consumer = new EventHubsConsumer();
-            consumer.Configure();
+            IReader reader = new EventReader();
+            reader.Configure();
 
             Console.WriteLine("Dequeuing messages...");
             var cts = new CancellationTokenSource();
             var task = Task.Factory.StartNew(MessageLoop,
```

Comment:
I'd recommend using `Task.Run` here; the options that you're setting aren't really needed and it adds complexity to an area that isn't related to what the sample is trying to demonstrate for readers.